00-dnaRnaConversion/index.js

```diff
@@ -30,4 +30,35 @@ function toRna(input) {
 }
 
 toRna('ACGTGGTCTTAA') === 'UGCACCAGAAUU'
- */
\ No newline at end of file
+ */
+
+
+/**
+ * This function converts the provided DNA Strand into its corresponding RNA Strand.
+ * If there is an invalid letter provided in the string, an error message will be 
+ * returned ot the user.
+ * @param {input} DNA Strand
+ * @returns Corresponding RNA Strand
+ */
+function toRna(input) {
+   var result = "";
```

Comment:
Please refrain from using `var` from now on. Use `let` or `const` from ES6 instead.

https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/